python/ray/train/lightning/lightning_trainer.py

```diff
@@ -541,7 +541,15 @@ def _lightning_train_loop_per_worker(config):
 
     # Restore from a previously failed run
     checkpoint = session.get_checkpoint()
-    if checkpoint and "ckpt_path" not in trainer_fit_params:
+    if checkpoint:
+        logger.info("Restoring LightningTrainer from a failed run. ")
```

Comment:
Actually is this always true? What happens if the user passes in an AIR checkpoint?

```
LightningTrainer(..., resume_from_checkpoint=some_checkpoint)
```

Reply:
It might not be a failed run. Updated the log message.